roles/microshift_generate_iso/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+microshift_generate_iso_action: "install"
+microshift_generate_iso_action_allowed: ['install']
```

Comment:
if this is the only action allowed why even make it dynamic?

Reply:
I think this is part of the feedback I provided. By doing this it avoids injecting any value into `microshift_generate_iso_action` allowing only to load controlled "actions", for now is only install.yml, and not loading any other file.

Reply:
I simply wish to introduce another option like the cleanup. If necessary, I can remove it for now.